atm.js

```diff
@@ -1,3 +1,106 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+	var userAccounts = {
```

Comment:
Nice! What are other ways you could incorporate data/functionality into an object?